src/Commands/Env/CloneContentCommand.php

```diff
@@ -43,6 +46,8 @@ class CloneContentCommand extends TerminusCommand implements SiteAwareInterface
      * @option bool $db-only Only clone database
      * @option bool $files-only Only clone files
      * @option bool $updatedb Update the Drupal database
+     * @option string $search-for URL to search for (wordpress only)
+     * @option string $replace-with URL to replace (wordpress only)
```

Comment:
`--from_url` `--to_url` I think to keep consistent with internal API

Reply:
Done.